### PR TITLE
docs: add MAINTAINING.md

### PR DESCRIPTION
**Pull request by mrice32**
**Motivation**

To keep the repos organized, we need to add an on-duty maintainer structure.

**Summary**

Added `MAINTAINING.md`.

**Testing**

Check a box to describe how you tested these changes and list the steps for reviewers to test.

- [ ]  Ran end-to-end test, running the code as in production
- [ ]  New unit tests created
- [x]  Existing tests adequate, no new tests required
- [ ]  All existing tests pass
- [ ]  Untested

**Issue(s)**

N/A

### CODE REVIEW

**Review by nicholaspai — APPROVED** [review]
LGTM. Is there a way for GH to manage the rotation of the on-call maintainer? This way an external PR could request the "maintainer" role and this would auto select the on call person.

File: MAINTAINING.md
Comment (chrismaree) [inline MAINTAINING.md:9]: should this not just apply to the protocol repo? the rules & process on the UMIP/front end repo will properly need to be different, don't you think?

File: MAINTAINING.md
Comment (chrismaree) [inline MAINTAINING.md:1]: do you want to add a section that mentions how we cycle between weeks of on-call for maintaining? i.e "each eng is on-call for one week in a round robbin fashion" or should this be left off gh?

**Review by chrismaree — APPROVED** [review]
Looks good! had 2 questions but otherwise I'm happy :)

**Comment by mrice32** [comment]
> LGTM. Is there a way for GH to manage the rotation of the on-call maintainer? This way an external PR could request the "maintainer" role and this would auto select the on call person.

@nicholaspai That would be so cool. Will look into this.

File: MAINTAINING.md
Comment (mrice32) [reply MAINTAINING.md:9]: It's a good question. Maybe it shouldn't apply to UMIPs repo, but my feeling is that it _should_ apply to EMP-tools, EMP-launch, docs, etc? Maybe we just mention the exceptions here? Or do you think there's a better way to organize it?

**Comment by mrice32** [comment]
> > LGTM. Is there a way for GH to manage the rotation of the on-call maintainer? This way an external PR could request the "maintainer" role and this would auto select the on call person.
> 
> @nicholaspai That would be so cool. Will look into this.

@nicholaspai this might do the trick: https://github.com/marketplace/actions/team-rotation

File: MAINTAINING.md
Comment (Tommy1231232) [reply MAINTAINING.md:9]: We have a separate process managing the UMIP repo so it makes sense to exclude it. I think all of the other repos are applicable. E.g., if it's a doc PR then tag a non-eng to work on it

File: MAINTAINING.md
Comment (Tommy1231232) [inline MAINTAINING.md:11]: It seems like the ENG team being assigned to PRs and making sure they are continually moving forward is time consuming and takes away from items in the existing sprint. Would it be useful to track the work ENG is doing on these PRs in our existing sprint?

Comment (mrice32) [reply MAINTAINING.md:11]: It definitely takes time, but I'm not sure how useful it will be to explicitly track the review work in the sprint. I'm definitely open to the idea, though -- what sort of tracking process would you propose?

File: MAINTAINING.md
Comment (mrice32) [reply MAINTAINING.md:1]: I don't know if it should be managed off-github, but I think this doc shouldn't prescribe the rotation. IMO, that should be managed in whatever tool/doc we use to actually do the rotation. I think this doc should just be able to point to somewhere you can go to figure out who the current on-call maintainer is.

**Comment by pemulis** [comment]
Anything we're waiting on here before merging @mrice32?

File: MAINTAINING.md
Comment (chrismaree) [reply MAINTAINING.md:9]: ye, I think it best to scope this to just protocol for now.

**Pull request merged**
